packages/expo-standard-web-crypto/package.json

```diff
@@ -31,13 +31,13 @@
   "license": "MIT",
   "devDependencies": {
     "expo-module-scripts": "^3.0.0",
-    "expo-random": "~13.1.0"
+    "expo-crypto": "^12.2.1"
   },
   "peerDependencies": {
-    "expo-random": "13.1.1"
+    "expo-crypto": "12.2.1"
```

Comment:
@EvanBacon Maybe the peer dependency of `expo-crypto` should be more permissive like `^12.2.1` to avoid conflict with other packages ?